src/app-layer-ssh.c

```diff
@@ -55,6 +55,8 @@
 
 /* HASSH fingerprints are disabled by default */
 #define SSH_CONFIG_DEFAULT_HASSH false
+/* Bypassing the encrypted part of the connections */
+#define SSH_CONFIG_DEFAULT_ENCRYPTION_BYPASS SSH_HANDLE_ENCRYPTION_DEFAULT
```

Comment:
nit : why do we need this define instead of reusing `SSH_HANDLE_ENCRYPTION_DEFAULT` directly ?

Reply:
Because the default is supposed to mean the actual action (as you highlighted stop_inpection_only)  to take but considering it is only used in one code place, I can use it there. 
I also took the same template as with the above SSH_CONFIG_DEFAULT_HASSH where it has the value assigned directly.